pyogrio/tests/test_geopandas_io.py

```diff
@@ -704,3 +704,26 @@ def test_custom_crs_io(tmpdir, naturalearth_lowres_all_ext):
     assert crs["lat_2"] == 51.5
     assert crs["lon_0"] == 4.3
     assert df.crs.equals(expected.crs)
+
+
+@pytest.mark.parametrize("ext", ALL_EXTS)
+def test_write_read_null(tmp_path, ext):
+    from shapely.geometry import Point
+
+    output_path = tmp_path / f"test_write_nan.{ext}"
+    geom = Point(0, 0)
+    test_data = {
+        "geometry": [geom, geom, geom],
+        "float64": [1.0, None, np.nan],
```

Comment:
This doesn't have any effect in practice, as that will get converted to twice a NaN by pandas:

```
In [10]: pd.Series([1.0, None, np.nan])
Out[10]: 
0    1.0
1    NaN
2    NaN
dtype: float64
```

Reply:
Yes, that's indeed the case. Nonetheless I think it is transparant that it is explicitly in the test?

But obviously if you think that's better I can put 2 times np.nan or whatever...